inc/Engine/Optimization/RUCSS/Database/Queries/UsedCSS.php

```diff
@@ -578,6 +579,46 @@ private function table_exists(): bool {
 		return $exists;
 	}
 
+	/**
+	 * Update the error message.
+	 *
+	 * @param int    $job_id Job ID.
+	 * @param int    $code Response code.
+	 * @param string $message Response message.
+	 * @param string $previous_message Previous saved message.
+	 *
+	 * @return bool
+	 */
+	public function update_message( int $job_id, int $code, string $message, string $previous_message = '' ): bool {
+		return $this->update_item(
+			$job_id,
+			[
+				'error_message' => $previous_message . ' - ' . current_time( 'mysql', true ) . " {$code}: {$message}",
+			]
+		);
+	}
+
+	/**
+	 * Updates the not_processed_before field
+	 *
+	 * @param mixed      $job_id the job id.
+	 * @param string|int $not_processed_before timestamp or mysql format date.
+	 *
+	 * @return bool either it is save or not.
+	 */
+	public function update_not_processed_before( $job_id, $not_processed_before ): bool {
+		if ( is_numeric( $not_processed_before ) ) {
+			$not_processed_before = gmdate( 'Y-m-d H:i:s', $not_processed_before );
```

Comment:
I'm not sure what is the exact type of this column, is it timestamp or a date?
Because below in the row class we are converting it using strtotime and here we are changing it to date and then in `DefaultProcess` class we are dealing with it as timestamp, I believe we can easily keep all of them as a timestamp, what do u think?

Reply:
I've changed this function to save it as a timestamp, as it is a timestamp in the database.